storage/smiles/src/main/java/org/openscience/cdk/smiles/SmiFlavor.java

```diff
@@ -172,6 +172,13 @@ private SmiFlavor() {
      */
     public static final int Absolute            = Canonical | Isomeric;
 
+
+    /**
+     * Default SMILES output write Stereochemistry, Atomic Mass, and CXSMILES layers. The
+     * ordering is not canonical.
+     */
+    public static final int Default             = Stereo | AtomicMass | CxSmiles;
```

Comment:
I need to read more about this (and we need a sentence + reference in https://github.com/cdk/cdk-paper-3), but is CXSMILES something people would expect as default?